pages/docs/reference/idioms.md

```diff
@@ -118,16 +118,17 @@ val list = listOf("a", "b", "c")
 
 <div class="sample" markdown="1" theme="idea" data-highlight-only>
 ```kotlin
-val map = mapOf("a" to 1, "b" to 2, "c" to 3)
+val map = mapOf("key-a" to "value-a", "key-b" to "value-b")
```

Comment:
Perhaps we can add an explanation in a comment. This will keep the code concise. What do you think?
```suggestion
val map = mapOf("a" to 1, "b" to 2) // "key to value" pairs
```

Reply:
I think my solution is more understandable for beginners.